pypeit/spectrographs/magellan_mage.py

```diff
@@ -2,6 +2,9 @@
 """
```

Comment:
Currently the structure of this spectrograph files differs from the other fixed format echelle's. We need to make these the same, and ideally they would be children of an echelle spectrograph class, that has whatever extra methods are required.